reg/drone/service/actuator/util/Feedback.0.1.uavcan

```diff
@@ -0,0 +1,26 @@
+# This high-rate feedback should be published once immediately after a setpoint is applied.
+# It follows that the publication rate of these messages equals that of the setpoint messages.
+# When setpoint messages are not being emitted, the publication rate is implementation-defined.
+# The priority of this message should be the same as that of the corresponding setpoint message.
+
+float32 SETPOINT_EXPIRATION_TIMEOUT = 1.0  # [seconds]
+# The actuator is allowed to disarm/disengage itself (e.g., by entering a power-down mode, stopping the controlled
+# mechanism, disconnecting itself from the controlled entity, etc. depending on the specifics of the application
+# at hand) if no setpoint messages have been received in this amount of time.
+# In this case, the arming flag would be cleared.
+
+reg.drone.service.util.Health.0.1 health
```

Comment:
We should also consider the polymorphic Health sub-system that @thirtytwobits suggested

Reply:
Let us summon @thirtytwobits into this conversation and let him shed some light upon his idea. The service health type defined here has six unused bits that can be leveraged for service-specific health data, but I am not exactly clear on what could be the use case for that. For example, for ESC we already have the extra service-specific information added at the end of this type. Seeing as Health is positioned first and is `@final`, one can subscribe to this actuator feedback type using the Health type and receive correct data (implicit truncation yay).